src/dlgTriggerEditor.cpp

```diff
@@ -592,17 +610,14 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
     treeWidget_keys->hide();
     treeWidget_variables->hide();
 
-    popupArea->hide();
-    frame_rightBottom->hide();
-
     readSettings();
 
     treeWidget_searchResults->setColumnCount(4);
     QStringList labelList;
-    labelList << "Type"
-              << "Name"
-              << "Where"
-              << "What";
+    labelList << tr("Type", "Heading for the first column of the search results display treewidget")
```

Comment:
Perhaps it would be more precise to say: "Translators may not be coders?" :stuck_out_tongue_winking_eye: 

However, you probably do have a point there - OTOH I was using the term we use for the widget precisely because that is what we (well someone deep in the Mudlet past) decided to call it. I will cull it though.